docs/governance/politeia.md

```diff
@@ -19,7 +19,7 @@ There is a fee for submitting a proposal (0.1 DCR), to limit the potential for p
 
 Reddit-style **up/down voting** is used for **proposal and comment sorting only**. Up/down votes are not anonymous.
```

Comment:
up/down votes are not currently used for proposal sorting.  Just comment sorting.